R/package.R

```diff
@@ -182,18 +182,16 @@ check_bootstrap_version <- function(version, pkg) {
 pkgdown_config_path <- function(path) {
   path_first_existing(
     path,
-    c("_pkgdown.yml",
-      "_pkgdown.yaml",
-      "pkgdown/_pkgdown.yml",
-      "inst/_pkgdown.yml"
+    c(
+      "_pkgdown.yml", "_pkgdown.yaml",
+      "pkgdown/_pkgdown.yml", "pkgdown/_pkgdown.yaml",
+      "inst/_pkgdown.yml", "inst/_pkgdown.yaml"
     )
   )
 }
 pkgdown_config_href <- function(path) {
-  cli::style_hyperlink(
-    text = "_pkgdown.yml",
-    url = paste0("file://", pkgdown_config_path(path))
-  )
+  config <- pkgdown_config_path(path) %||% "_pkgdown.yml"
```

Comment:
why should `pkgdown_config_href()` return a styled hyperlink to a file that does not exist? Shouldn't it instead link to https://pkgdown.r-lib.org/articles/pkgdown.html#metadata?

Also, if the configuration file does not exist, shouldn't `check_pkgdown()` exit sooner?

Reply:
E.g. in `"Either add to {pkgdown_config_href({pkg$src_path})} or use @keywords internal"` it could be confusing to have the link to a file that does not exist.

Reply:
I should've noted that I only added that for tests; I don't think it'll come up in real life.

Reply:
why not add the config in the tests instead?

Reply:
Yeah good call.